proposed/2021/ShowVulnerabilitiesInThePMUI.md

```diff
@@ -0,0 +1,90 @@
+- Users can see when a package search result in the Browse tab has a known vulnerability, to help them make informed decisions when choosing packages.
+- Users can find which installed top level packages have a vulnerability, to help them take the appropriate action.
+- Minor changes to the deprecation UX to align with necessary vulnerability UX.
+
+The following scenarios will not be in the MVP, but will be considered for next iterations:
+
+- Users can discover if they have installed packages with vulnerabilities through indicators in the solution explorer.
+- Users can see which transitively installed packages have vulnerabilities.
+- Users have a 1-click or wizard experience to “fix” vulnerabilities.
+- Plumbing deprecation and vulnerability data into the search API for a performance improvement.
+- Users must explicitly confirm that they would like to install a deprecated or vulnerable package version.
+- Users can choose to sort their installed packages such that deprecated or vulnerable packages rise to the top.
+- All deprecation UX changes requested in Home/VSdeprecationfeature.md at dev · NuGet/Home (github.com).
+
+The following scenarios are considered separate from this feature but may considered in another spec:
+- Users can discover if they have installed packages with vulnerabilities through warnings in restore.
+- Users can discover if they have installed packages with vulnerabilities through warning messages or dialogues outside of the PMUI.
+
+## MVP Design
+
+- Like deprecation, a warning sign will appear on the Installed tab header when a top-level deprecated package is installed.
+    - The warning sign on the Installed tab header will display the number of vulnerable and deprecated packages in the tooltip.
+The warning icon will appear for all levels of vulnerabilities – consistent with the behavior on NuGet.org.
+    - A bolded warning will be shown at the bottom on the package list item to make it more noticeable when a package is deprecated or vulnerable.
+- We will use the same warning sign for vulnerabilities as we do for deprecation to avoid the “lucky charms” effect where symbols get ignored because there are too many.
+- Deprecated and vulnerable packages will automatically be shown at the top of the installed package list in alphabetical order. The rest of the packages will be below in alphabetical order.
+- The version dropdown list will show “(Vulnerable)” beside the relevant versions.
+    - If a package is vulnerable and deprecated, it will show “(Vulnerable, Deprecated)”
+- The package details window will display a more detailed vulnerability message that includes the severity and a link to the advisory details.
+    - If a package is both deprecated and has a vulnerability, we will display both detailed messages with the vulnerability message on top and the deprecation message below it.
+
+![image](https://user-images.githubusercontent.com/15097183/126406263-2aa24019-fe0f-43e0-a21a-c7fa41779a72.png)
```

Comment:
Please call out in the spec what the wording for the Installed tab tooltip should be since we decided to change it.